src/parser.cpp

```diff
@@ -1943,6 +1943,7 @@ gb_internal bool is_foreign_name_valid(String const &name) {
 			case '$':
 			case '.':
 			case '_':
+			case ',':
```

Comment:
Please don't just go an change a random procedure which is used by many other things, just to get this working. 

I will not accept this bodge.

Reply:
You're right, I should have considered the issue and filed an issue first. My apologies.

What can I do to help resolve the problem? I'll start by filing an issue that explains the problem and references here, and I'll close the PR.